HintList.py

```diff
@@ -589,17 +589,17 @@ def tokens_required_by_settings(world: World) -> int:
     'Water Temple MQ Freestanding Key':                            ("hidden in a #box under the lake# lies", "hidden in a #box in the Water Temple# lies", ['dungeon', 'sometimes']),
     'Water Temple MQ GS Freestanding Key Area':                    ("the #locked spider under the lake# holds", "the #locked spider in the Water Temple# holds", ['dungeon', 'sometimes']),
     'Water Temple MQ GS Triple Wall Torch':                        ("a spider behind a #gate under the lake# holds", "a spider behind a #gate in the Water Temple# holds", ['dungeon', 'sometimes']),
-    'Gerudo Training Ground Underwater Silver Rupee Chest':        (["those who seek #sunken silver rupees# will find", "the #thieves' underwater training# rewards"], None, ['dungeon', 'sometimes']),
-    'Gerudo Training Ground MQ Underwater Silver Rupee Chest':     (["those who seek #sunken silver rupees# will find", "the #thieves' underwater training# rewards"], None, ['dungeon', 'sometimes']),
-    'Gerudo Training Ground Maze Path Final Chest':                ("the final prize of #the thieves' training# is", None, ['dungeon', 'sometimes']),
-    'Gerudo Training Ground MQ Ice Arrows Chest':                  ("the final prize of #the thieves' training# is", None, ['dungeon', 'sometimes']),
+    'Gerudo Training Ground Underwater Silver Rupee Chest':        (["those who seek #sunken silver rupees# will find", "the #thieves' underwater training# rewards"], "obtaining #silver rupees drowned in Gerudo Training Grounds# rewards", ['dungeon', 'sometimes']),
+    'Gerudo Training Ground MQ Underwater Silver Rupee Chest':     (["those who seek #sunken silver rupees# will find", "the #thieves' underwater training# rewards"], "obtaining #silver rupees drowned in Gerudo Training Grounds# rewards", ['dungeon', 'sometimes']),
+    'Gerudo Training Ground Maze Path Final Chest':                ("the final prize of #the thieves' training# is", "#Gerudo Training Grounds' final reward# contains", ['dungeon', 'sometimes']),
```

Comment:
Maybe "The chest in the middle of Gerudo Training Grounds' maze"?

Reply:
I was asked to make the Ice Arrows chest in GTG the same text for both Normal and MQ.
Your description doesn't match that.

But I also feel it should be OK to separate these two.

Can you tell me which would be better in your opinion ?

Reply:
Ahh, that makes sense actually. Yeah, you can keep it as is